src/DSi.cpp

```diff
@@ -82,6 +82,14 @@ DSi_SDHost* SDIO;
 u64 ConsoleID;
 u8 eMMC_CID[16];
 
+// FIXME: these currently have no effect (and aren't stored in a savestate)
+//        ... not that they matter all that much
+u8 GPIO_Data;
```

Comment:
these registers don't seem to be set back again to 0 on Reset

Reply:
fixed (data actually resets to 0xff instead of 0, I've observed this with a scope and logic analyzer)